command/whitelist/deployment/params.go

```diff
@@ -0,0 +1,166 @@
+package deployment
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/0xPolygon/polygon-edge/chain"
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/0xPolygon/polygon-edge/command/helper"
+	"github.com/0xPolygon/polygon-edge/helper/common"
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+const (
+	chainFlag         = "chain"
+	addAddressFlag    = "addAddress"
+	removeAddressFlag = "removeAddress"
```

Comment:
These should be snake-case

Reply:
Why, never liked snake-case?
Fixed 4b273701cdbaefd586070c692b549c33cc187cdc

Reply:
You didn't fix this

Reply:
The flags should be in kebab-case:
`add-address` and `remove-address`

Reply:
my mistake a5782ff9d648ec62f1553c4e7174833f882854f8

Reply:
🙈 

The json types were alright to be snake_case....

Reply:
Ah, for the genesis they are...

Resolving this